docs/google-analytics.md

```diff
@@ -0,0 +1,70 @@
+## how google analytics works 
+
+Since angular is a single page application, i implented router checking. So when a user activates the Router function, it is recorded as visiting a new part of the page.
```

Comment:
```suggestion
Since angular is a single page application, I implemented router checking. So when a user activates the Router function, it is recorded as visiting a new part of the page.
```